src/app/containers/ComscoreAnalytics/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React, { useContext } from 'react';
+import { RequestContext } from '#contexts/RequestContext';
+import AmpComscoreAnalytics from './Amp';
+
+const ComscoreAnalytics = () => {
+  const { isAmp } = useContext(RequestContext);
```

Comment:
Did we want a toggle for this like we do with Chartbeat?

Reply:
Added!